### PR TITLE
Handle units containing 'degree' and 'per' correctly

### PR DESCRIPTION
**Pull request by nielstron**

### CODE REVIEW

File: tests/test_compounds.py [hunk 1]
```diff
@@ -2,19 +2,44 @@
...
+        'degree rankine': 'degrees rankine'
+    }
+    for singular, plural in test_cases.items():
+        eq_(p.plural(singular), plural)
+
+
+def test_unit_handling_fractional():
+    test_cases = {
+        'pound per square inch': 'pounds per square inch',
+        'pound-force per square inch': 'pound-forces per square inch',
```
[the remark this answers is not in the document]
Comment (nielstron) [reply tests/test_compounds.py]: No I think this is correct:
"pound-force (plural pound-forces)" from [Wiktionary](https://en.wiktionary.org/wiki/pound-force]

[the remark this answers is not in the document]
Comment (hugovk) [reply tests/test_compounds.py]: Here's the *Oxford English Dictionary*:

image

None of the other [10 dictionaries](https://www.onelook.com/?w=pound+force&ls=a&loc=2osdf) I checked show a plural form at all.

Google Books has [66 results](https://www.google.com/search?num=100&rlz=1C5CHFA_enFI726FI726&biw=1258&bih=698&tbm=bks&ei=F5isW5r_FqvJgAam05HwCw&q=%22pounds+force+per+inch%22&oq=%22pounds+force+per+inch%22&gs_l=psy-ab.3...125.1749.0.2012.9.9.0.0.0.0.110.735.7j2.9.0....0...1c.1.64.psy-ab..0.0.0....0.cwrPYbMNBUU) for "pounds force per inch".

But only [two results](https://www.google.com/search?num=100&rlz=1C5CHFA_enFI726FI726&biw=1258&bih=698&tbm=bks&ei=4ZisW9SbF8iUwAS0kIzQAw&q=%22pound+forces+per+inch%22&oq=%22pound+forces+per+inch%22&gs_l=psy-ab.3...53700.54495.0.54707.2.2.0.0.0.0.122.180.1j1.2.0....0...1c.1.64.psy-ab..0.0.0....0.GJxJca2jXT4) for "pound forces per inch".

The plural form on Wiktionary was added when the `en-noun` template was simply added, without taking care of the special-case plural:

```
{{en-noun|head=[[pound]]-[[force]]}}
```

https://en.wiktionary.org/w/index.php?title=pound-force&diff=13830692&oldid=13487437

I've fixed Wiktionary.

[the remark this answers is not in the document]
Comment (hugovk) [reply tests/test_compounds.py]: Metric is so much easier :)

File: tests/test_compounds.py [hunk 2]
```diff
@@ -37,9 +37,21 @@ def test_unit_handling_fractional():
...
+    }
+    for singular, plural in test_cases.items():
+        eq_(p.plural(singular), plural)
+
+
+def test_unit_handling_combined():
+    test_cases = {
+        # Heat transfer coefficient unit
+        'watt per square meter degree celsius': 'watts per square meter degree celsius',
+        'degree celsius per hour': 'degrees celsius per hour'
```
Comment (hugovk) [inline tests/test_compounds.py]: Big C in "Celsius"

Comment (nielstron) [reply tests/test_compounds.py]: This needs #55 then to be merged into master to work.

**Pull request merged**